plugins/org.eclipse.elk.alg.layered/src/org/eclipse/elk/alg/layered/p5edges/orthogonal/HyperEdgeSegment.java

```diff
@@ -152,7 +154,7 @@ private static void insertSorted(final List<Double> list, final double value) {
 
     /////////////////////////////////////////////////////////////////////////////////////////////////////////////////
     // Getters and Setters
-
+    
```

Comment:
Trailing whitespace

Reply:
Personally, I don't think trailing whitespace is enough of an issue to go through everything again. However, I did configure Eclipse now to remove trailing whitespace on save. 🙂